projects/storefrontlib/src/shared/utils/forms/form-utils.ts

```diff
@@ -29,9 +29,7 @@ export class FormUtils {
    * @param form Form with fields to mark
    */
   static markFormAsTouched(form: FormGroup): void {
-    Object.keys(form.controls).forEach(controlName => {
-      form.get(controlName).markAsTouched();
-    });
+    form.markAllAsTouched();
```

Comment:
Now that we have this simplified code, can we just drop the `markFormAsTouched` method?

Reply:
removed